src/txn/txn_ckpt.c

```diff
@@ -619,19 +617,6 @@ __checkpoint_prepare(WT_SESSION_IMPL *session, bool *trackingp, const char *cfg[
 
     __wt_writeunlock(session, &txn_global->rwlock);
 
-    if (F_ISSET(txn, WT_TXN_HAS_TS_READ)) {
-        __wt_verbose_timestamp(
```

Comment:
We still need to have this verbose msg.

Reply:
Added it back, it will still log if the stable is 0 which to me seems like correct behaviour.